### DISCUSSION TITLE
Twitch store redemptions stopped appearing in new updates

### DISCUSSION THREAD

**Discussion by Graulybr** [baffler/Transparent-Twitch-Chat-Overlay#243] [General]
I updated the program and noticed the visual improvements and functionality that the program had.
I was very happy to learn that the Twitch chat overlay is still in development.
Exit the 0.9.4 upgrade to 1.0.2!
In version 0.9.4 the program worked correctly and with all the correct functions, but after I upgraded I noticed that the redemptions stopped appearing for me.
At first I thought I had made a mistake in some configuration.
But I soon realized that this was not the case, to be sure I decided to go back to version 0.9.4 and everything went back to normal.

I would like to know if this is just me or has anyone else faced this problem!
I hope the problem is resolved, in the meantime I will be using version 0.9.4

**Comment by VectorRar** [comment]
It isn't working for me either

**Comment by baffler** [comment]
This is fixed with the latest version [v1.0.4](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v1.0.4)
